operate/client/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@axe-core/playwright": "4.10.0",
     "@bpmn-io/element-template-icon-renderer": "0.5.2",
-    "@camunda/camunda-composite-components": "0.9.0",
+    "@camunda/camunda-composite-components": "0.9.1-rc.2",
```

Comment:
I think we might need to wait for the patch/minor version to be released before merging this.
Can you please review it and once this is available, i will update the version here and merge it.